ui/src/app/pages/admin/admin-banner-modal.spec.tsx

```diff
@@ -207,3 +208,39 @@ describe('AdminBannerModal', () => {
     });
   });
 });
+
+describe('AdminBannerModal - form validation', () => {
+  it('returns no errors for completed form', () => {
+    // Arrange
+    const filledBanner: StatusAlert = {
+      title: 'Test Title',
+      message: 'Test Message',
+      link: '',
+      alertLocation: StatusAlertLocation.AFTER_LOGIN,
+      startTimeEpochMillis: Date.now(),
+    };
+
+    // Act
+    const errors = validateBannerAlert(filledBanner);
+
+    // Assert
+    const expectedErrors: Record<string, string[]> | undefined = undefined;
+    expect(errors).toEqual(expectedErrors);
+  });
+
+  it('returns errors empty form', () => {
+    // Arrange
+    const filledBanner: StatusAlert = {} as Partial<StatusAlert> as StatusAlert;
+
+    // Act
+    const errors = validateBannerAlert(filledBanner);
+    // Assert
+    const expectedErrors: Record<string, string[]> = {
+      title: ['Please enter a banner title'],
+      message: ['Please enter a banner message'],
+      startTimeEpochMillis: ['Please enter a start time'],
+      alertLocation: ['Please select a banner location'],
+    };
+    expect(errors).toEqual(expectedErrors);
```

Comment:
<img width="503" alt="image" src="https://github.com/user-attachments/assets/af2b4c71-8e8d-4fb1-a271-a9eb89ec6eeb" />

I'm not sure if there is a way to trigger the alertLocation validation message from within the UI.